README.adoc

```diff
@@ -0,0 +1,117 @@
+= The Agriculture Core
+
+_The Agriculture Core_ is a NodeJS server exposing a robust REST API for various client applications to connect to.
+
+== Configuration
+
+The agriculture-core pulls its configuration from a `.env` file at the root of the project.
+These configuration variables are loaded into `process.env.<VAR_NAME>` and are accessible accross the application.
+To get started, this configuration file must be created with the following format:
+
+[source,properties]
+----
+# Application Configuration
+PORT=
```

Comment:
# Requested Favour

This is absolutely my fault from when I made the README originally, I forgot to add a `LOG_LEVEL` option. This has the option of:
```
{ 
  error: 0, 
  warn: 1, 
  info: 2, 
  verbose: 3, 
  debug: 4, 
  silly: 5 
}
```
And will control which logs are generated